src/gui/MacOSXBundleInfo.plist

```diff
@@ -34,6 +34,8 @@
         <string>F9ZGRXJE7XbQb2Kt36hwaBO4rYjkXYHiNS5hd+MkyKY=</string>
         <key>SUAllowsAutomaticUpdates</key>
         <string>NO</string>
+        <key>LSMinimumSystemVersion</key>
+        <string>${MACOSX_DEPLOYMENT_TARGET}</string>
```

Comment:
Any reason not to use `CMAKE_OSX_DEPLOYMENT_TARGET` directly?

Reply:
No, but I want to first look at the build artifacts and see if this works